lib/tasks/spokeo.rake

```diff
@@ -0,0 +1,13 @@
+require "httpx"
+require "pry"
+
+namespace :spokeo do
+  task search: :environment do |_|
+    state = "florida"
+    name = "my name"
+
+    search_result = Spokeo::Search.run(name: name, state: state)
```

Comment:
So you're never passing in `city` here?

Reply:
Nah, not here. I had it initially to try and drill down results. I think we'll need the functionality very soon.